contrib/platform/src/com/sun/jna/platform/unix/X11.java

```diff
@@ -795,6 +795,8 @@ void XSetWMProperties(Display display, Window window, String window_name,
                           String icon_name, String[] argv, int argc,
                           XSizeHints normal_hints, Pointer wm_hints,
                           Pointer class_hints);
+
+    int XSetWMProtocols(Display display, Window window, Atom[] atom, int count);
```

Comment:
This is a nice to have: Should the XGetWMProtocols function also directly implemented (looks a bit more symmetrical :-))